src/sql/workbench/contrib/notebook/browser/outputs/renderers.ts

```diff
@@ -338,10 +338,7 @@ export function renderSVG(options: renderSVG.IRenderOptions): Promise<void> {
 		return Promise.resolve(undefined);
```

Comment:
I happened to also notice that the text on line 337 isn't localized - while you're changing this maybe want to fix that as well?